meta-oe/recipes-extended/tmate/tmate_2.4.0.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Instant terminal sharing"
+DESCRIPTION = "Tmate is a fork of tmux. It provides an instant pairing solution."
+AUTHOR = "Marius Kriegerowski <marius.kriegerowski@gmail.com>"
+HOMEPAGE = "https://tmate.io/"
+LICENSE = "BSD"
```

Comment:
see which particular BSD license it is. I think it also has Expat in there so I wonder if Expat should  be mentioned or not.

Reply:
They use the `ISC` license. Changed the `LICENSE` filed accordingly.